Automattic/wp-calypso
# Client: separate out @format from external deps block

**The Problem**
For two different reasons it is less than optimal for us to combine our dependencies docblocks and the format pragma docblock:
1. currently eslint recognizes it as an error (can be separately addressed)
2. our selectors devdocs are picking it up and treating it like a parameter (can also be separately addressed by improving the parser).  Here is a screenshot example:
<img width="962" alt="screen shot 2017-11-13 at 12 49 36 pm" src="https://user-images.githubusercontent.com/4656974/32740502-2c7115c0-c871-11e7-8d75-9254d4b61aef.png">

---

**Solution**
Sublime's multi-line find and replace to convert these two blocks:
```
/**
 * Internal dependencies
 *
 * @format
 */
```
```
/**
 * External dependencies
 *
 * @format
 */
```

Into these two blocks
```
/** @format */

/**
 * Internal dependencies
 */
```
```
/** @format */

/**
 * External dependencies
 */
```

It is case insensitive.  Heres a screenshot example:
<img width="1440" alt="screen shot 2017-11-13 at 12 40 25 pm" src="https://user-images.githubusercontent.com/4656974/32740309-8d091fb4-c870-11e7-9ea0-0bddd2fc7424.png">


To Test
-----
1. `npm start` and slight smoke test.
2. check devdocs and see that selectors no longer think `@format` is a parameter for them
